BloomFilter.cpp

```diff
@@ -1,41 +1,40 @@
 #include "BloomFilter.h"
+#include <bitset>
 
 template <typename T>
 BloomFilter<T>::BloomFilter(){
   this->m = 10;
   this->k = 3;
-
-  this->bit_arr = new int[m];
+  bitset<10> b;
+  /*bitset<num> is a template and num needs to be known at compile time*/
+  this->bit_arr = b;
   this->hash_fxn = hash_fxn;
 }
```

Comment:
I think this should be a one-liner:

this(10, 3, DEFAULT_HASH_FUNCTION);  // < default hash function needs to be defined